### ISSUE TITLE
Add Disboard

### ISSUE THREAD

**Issue by connor7000** [mc-ascendancy/ascendancy#4]
From **SL#6969** on Discord:

"Add Disboard so we get more players."

**Comment by connor7000** [comment]
Certainly, this set-up / configuration shouldn't be too tough, right @brussels-sprout?

I'm personally interested as to how the DISBOARD ecosystem has treated Discord server dedicated to Minecraft servers in the past. I suppose there's no harm in getting this implemented-- just to try it-- though!

**Comment by brussels-sprout** [comment]
It'd be very easy to add DISBOARD and I don't see why not. So should I add it then, @connor7000?
